src/test/java/choorai/excuseme/lecture/ui/LectureControllerTest.java

```diff
@@ -47,14 +53,75 @@ void afterEach() {
         memberRepository.deleteAll();
     }
 
+    @Test
+    @DisplayName("전체 강의 목록을 조회한다.")
+    void find_all_lectures() throws Exception {
+        // given
+        final Member normalMember = Member.createNormalMember(
+            "a@email.com",
+            passwordEncoder.encode("password1@"),
+            "이름",
+            "MEN",
+            "19990101",
+            "01012345678"
+        );
+        memberRepository.save(normalMember);
+
+        IntStream.rangeClosed(1, 10).forEach(i -> {
+            lectureRepository.save(new Lecture("lecture" + i, "thumbnail" + i, "videoUrl" + i));
+        });
+
+        final String accessToken = jwtProvider.createToken("a@email.com", Role.USER);
+
+        // when
+        final ExtractableResponse<Response> response = RestAssured
+            .given().headers("Authorization", "Bearer " + accessToken)
+            .when().get("/lectures")
+            .then().statusCode(HttpStatus.OK.value())
+            .extract();
```

Comment:
p2

```suggestion
        final List<LectureResponse> response = RestAssured
            .given().headers("Authorization", "Bearer " + accessToken)
            .when().get("/lectures")
            .then().statusCode(HttpStatus.OK.value())
            .extract().jsonPath().getList(" ", LectureResponse.class);
```

여기도 이렇게 변경하는 것이 더 깔끔할 것 같습니다!

Reply:
감사합니다! 바꿔보겠습니다! ㅎㅎ